src/server/resolve.rs

```diff
@@ -95,9 +108,20 @@ where
             // it explicitly.
             Err(pd_client::Error::StoreTombstone(_)) => {
                 RESOLVE_STORE_COUNTER_STATIC.tombstone.inc();
-                return Err(box_err!("store {} has been removed", store_id));
+                self.router.report_store_maybe_tombstone(store_id);
```

Comment:
should avoid calling it too frequent as `broadcast_normal` is a quite time-consuming operation holding lock which would affect raftstore

Reply:
For now, it's a blank implementation for raftstore v1, and only works for raftstore v2 which has fewer regions. It might be okay (?) I think. I will add a metric to mentoring the cost.

Reply:
Turns out there is already a metric for broadcast duration, `tikv_broadcast_normal_duration_seconds`.